js/hw9_1.js

```diff
@@ -0,0 +1,95 @@
+// task1
+
+// building HTML
+
+// wrapper for entire block
+$(".farcry__section").append('<article class="card"></article>');
+
+// navigation
+$(".card").append('<ul class="nav"></nav>');
+$(".nav").append(`<li class="nav-item nav-item--active nav-item__info">
```

Comment:
This type of adding dynamic HTML is very poor ^_^

Reply:
Probably, but it's compact and easy to understand. Also it's one of the options from official documentation https://api.jquery.com/append/.

Reply:
Agree. But imagine if you have a big HTML to add and it would be hard to maintain it. Would be easier to create a variable and edit it in that place